model/collection.go

```diff
@@ -4,7 +4,17 @@
 package model
 
 type CollectionMetadata struct {
-	Id          string `json:"id"`
-	Name        string `json:"name"`
-	RelativeURL string `json:"url"`
+	Id             string `json:"id"`
+	CollectionType string `json:"collection_type"`
+	TeamId         string `json:"team_id"`
+	Name           string `json:"name"`
+	RelativeURL    string `json:"relative_url"`
```

Comment:
nit, suggested order (Id > TeamId > *Type)
```suggestion
	Id             string `json:"id"`
	TeamId         string `json:"team_id"`
	CollectionType string `json:"collection_type"`
	Name           string `json:"name"`
	RelativeURL    string `json:"relative_url"`
```